core/start-blueos-core

```diff
@@ -148,9 +149,56 @@ function create_service {
     done
 
     # Use run_service to start the service with the memory limit
-    tmux send-keys -t $SESSION_NAME "run-service '$command' $memory_limit_mb" C-m
+    tmux send-keys -t $SESSION_NAME "run-service '$SERVICE_NAME' '$command' $memory_limit_mb " C-m
 }
 
+ssh_command() {
+  ssh -i /root/.config/.ssh/id_rsa -o StrictHostKeyChecking=no pi@localhost "$1"
+}
+
+prepare_cgroups() {
+  # Prepare cgroups for run-service.sh to use
+  # Get the Docker container's hash by its name
+  FULL_COMMAND="docker ps --no-trunc --filter name=blueos-core --format '{{.ID}}'"
+  CONTAINER_HASH=$(ssh_command "$FULL_COMMAND")
+  # Use the Docker container's hash to generate the path
+  FULL_PATH="system.slice/docker-$CONTAINER_HASH.scope"
+
+  # Remove the starting '0::' from FULL_PATH
+  FULL_PATH=${FULL_PATH#0::}
+  # Remove the leading '/'
+  FULL_PATH=${FULL_PATH#/}
+  echo "Docker cgroups root: $FULL_PATH"
+  # Export the full path
+  export DOCKER_CGROUP=$FULL_PATH
+
+  # populated cgroups cannot be changed, so, in order to enable subtree_control, 
+  # we need to create a new child cgroup andmove all processes of this cgroup to it
+  # (as only empty cgroups can be changed).
+
+  echo "Creating a new child cgroup..."
+  # Create a new child cgroup
+  mkdir /sys/fs/cgroup/$DOCKER_CGROUP/child_cgroup
```

Comment:
We should create a variable for /sys/fs/cgroup/$DOCKER_CGROUP, since it's used so many times

Reply:
done